src/java.base/share/classes/javax/crypto/ExemptionMechanism.java

```diff
@@ -71,7 +71,7 @@ public class ExemptionMechanism {
     private Key keyStored = null;
 
     /**
-     * Creates a ExemptionMechanism object.
+     * Creates an <code>ExemptionMechanism</code> object.
```

Comment:
Use `{@code }` instead of `<code></code>` for all of the javadoc?

Reply:
Yes.